lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,27 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let kTypeStrippingMode = null;
+ObjectDefineProperty(process.features, 'typescript', {
```

Comment:
Do we need a getter? It seems it should be a classic value like the other ones

Reply:
If you try to get the option value outside the getter, it throws:

```
Error: Should not query options before bootstrapping is done
```

Reply:
Cannot use `GetOption` before bootstrap so it needs a getter

Reply:
Let's add a comment then